pkg/chains/signing.go

```diff
@@ -223,10 +234,20 @@ func (o *ObjectSigner) Sign(ctx context.Context, tektonObj objects.TektonObject)
 	if err := MarkSigned(ctx, tektonObj, o.Pipelineclientset, extraAnnotations); err != nil {
 		return err
 	}
-
+	logger.Debugf("Recording count of marked payload for %v", tektonObj)
+	measureMetrics(MarkedAsSignedCount, o.Recorder, ctx)
 	return nil
 }
 
+func measureMetrics(metrictype string, mtr MetricsRecorder, ctx context.Context) {
+	logger := logging.FromContext(ctx)
+	logger.Debugf("Type of Metrics Recorder ", mtr, metrictype)
```

Comment:
I think there's too much logging happening. Can we remove this one?

Reply:
Removed